test/e2e/scaffold/k8s.go

```diff
@@ -106,6 +107,26 @@ func (s *Scaffold) ListPodsByLabels(labels string) ([]corev1.Pod, error) {
 	})
 }
 
+func (s *Scaffold) ListApisixSsl() ([]*v1.Ssl, error) {
```

Comment:
You can refer to it.

https://github.com/api7/api7-ingress-controller/blob/release-v2-dev/test/e2e/adminapi/dashboard_api.go#L168

Reply:
Not used, please delete it.